admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.firebase.example;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.UserProvider;
+import com.google.firebase.auth.UserRecord;
+
+public class FirebaseAuthSnippets {
+
+  public void getUserByFederatedId() throws Exception {
+    // [START get_user_by_federated_id]
+    UserRecord user = FirebaseAuth.getInstance().getUserByFederatedIdAsync("google.com", "google_uid1234").get();
```

Comment:
This should be getUserByProviderUidAsync (similar to the other ports). Yeah, I see that the upstream PR where you got this from is also incorrect and have flagged that.